.gitlab-ci.yml

```diff
@@ -92,6 +92,9 @@ deploy_live:
     - pip install boto3==1.14.47
     - pip install ecs-deploy==1.11.0
     - aws ssm get-parameters-by-path --region $AWS_DEFAULT_REGION --path /live/alegre/ --recursive --with-decryption --output text --query "Parameters[].[Name]" | sed -E 's#/live/alegre/##' > env.live.names
+    - for NAME in `cat env.live.names`; do echo -n "-s live-alegre-sbert $NAME /live/alegre/$NAME " >> live-alegre-sbert.env.args; done
+    - echo -n "-s live-alegre-sbert GITHUB_TOKEN arn:aws:secretsmanager:eu-west-1:848416313321:secret:GithubToken-Plain-BUhwIw" >> live-alegre-sbert.env.args
+    - ecs deploy ecs-live  live-alegre-sbert --diff --image live-alegre-sbert $QA_ECR_API_BASE_URL:$CI_COMMIT_SHA --timeout 1200 --exclusive-env -e live-alegre-sbert APP alegre -e live-alegre-sbert DEPLOY_ENV live -e live-alegre-sbert ALEGRE_PORT 8000 --exclusive-secrets `cat live-alegre-sbert.env.args`
```

Comment:
@sonoransun is this variable `$QA_ECR_API_BASE_URL` correct or it should be `$LIVE_ECR_API_BASE_URL`?

Reply:
Good catch! That should be live. I am fixing now :)